DaangnMarket/DaangnMarket/Source/Screen/Home1/Home1ViewController.swift

```diff
@@ -7,10 +7,108 @@
 
+import SnapKit
+import Then
+
 class Home1ViewController: UIViewController {
 
+    private lazy var tableView = UITableView().then {
+        $0.separatorColor = Color.daangnGray1
+        $0.separatorInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
+    }
+    
+    private let headerView = UIView().then {
+        $0.backgroundColor = .white
+    }
+    
+    private let sortButton = UIButton(type: .custom).then {
+        $0.titleLabel?.font = .systemFont(ofSize: 12)
+        $0.setTitle("추천순", for: .normal)
+        
+        $0.setTitleColor(Color.daangnGray3, for: .normal)
+        $0.setImage(Image.sortIcon, for: .normal)
+        
+        let spacing: CGFloat = 4
+        $0.imageEdgeInsets = UIEdgeInsets(top: 0, left: -spacing, bottom: 0, right: spacing)
+        $0.titleEdgeInsets = UIEdgeInsets(top: 0, left: spacing, bottom: 0, right: -spacing)
+    }
+    
+    private let filterButton = UIButton().then {
+        $0.titleLabel?.font = .systemFont(ofSize: 12)
+        $0.setTitle("가격", for: .normal)
+        
+        $0.setTitleColor(Color.daangnGray3, for: .normal)
+        $0.setImage(Image.filterIcon, for: .normal)
+        
+        let spacing: CGFloat = 4
+        $0.imageEdgeInsets = UIEdgeInsets(top: 0, left: -spacing, bottom: 0, right: spacing)
+        $0.titleEdgeInsets = UIEdgeInsets(top: 0, left: spacing, bottom: 0, right: -spacing)
+    }
+
     override func viewDidLoad() {
         super.viewDidLoad()
+        setLayouts()
+        setDelegation()
+    }
+}
+
+extension Home1ViewController {
+    private func setDelegation() {
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+}
 
+extension Home1ViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        return headerView
+    }
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 20
+        
+    }
+}
+
+extension Home1ViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 5
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        return UITableViewCell()
+    }
+}
+
+extension Home1ViewController {
+    func setLayouts() {
+        setViewHierarchies()
+        setConstraints()
+    }
+    
+    func setViewHierarchies() {
+        view.addSubviews(tableView)
+        headerView.addSubviews(sortButton)
+        headerView.addSubviews(filterButton)
+    }
+    
+    func setConstraints() {
+        tableView.snp.makeConstraints{
+            $0.top.bottom.equalToSuperview()
+            $0.leading.trailing.equalToSuperview().inset(16)
+        }
+        
+        filterButton.snp.makeConstraints{
+            $0.trailing.equalToSuperview().inset(16)
+            $0.bottom.equalTo(headerView.snp.bottom).inset(10)
```

Comment:
```suggestion
            $0.centerY.equalTo(headerView.snp.centerY)
```
중심으로 잡아주는게 깔끔할 것 같아요~~

Reply:
옛설 !~~~